ern-core/src/GitManifest.ts

```diff
@@ -232,12 +232,12 @@ export default class GitManifest {
       let pluginScope
       let pluginName
       let basePluginPath
-      if (scopeNameRe.test(plugin.basePath)) {
-        pluginScope = scopeNameRe.exec(plugin.basePath)![1]
-        pluginName = scopeNameRe.exec(plugin.basePath)![2]
+      if (scopeNameRe.test(plugin.name!)) {
+        pluginScope = scopeNameRe.exec(plugin.name!)![1]
+        pluginName = scopeNameRe.exec(plugin.name!)![2]
         basePluginPath = path.join(pluginsConfigurationDirectory, pluginScope)
       } else {
-        pluginName = plugin.basePath
+        pluginName = plugin.name!
```

Comment:
What exact does it mean that these properties now have a `!` at the end (compared to the previous behavior)?

Reply:
Does it mean the property can't be null?

Reply:
Yes, because `name` can be undefined (in the case of packages with a git path), TypeScript will complain. This is just to tell TypeScript "I know this property can be undefined under certain circumstances, but in the context of this code, it will never be undefined" (in this specific case, we do not support plugins -native modules- coming from git yet, only direct registry package paths, so `name` will always be defined).
It also answers your question in the comment you left, regarding how git `PackagePath` will be handled given that `name` will be undefined. TypeScript will complain every time we use `name` at locations where it cannot be `undefined`, so at least we are made aware that `name` can be undefined in certain cases and react accordingly.